README.md

```diff
@@ -25,21 +25,29 @@ Example:
       token: 'YOUR_TOKEN',
       endpoint: 'http://localhost:8080/api/v1',
       launchName: 'This is awesome',
-      launchDescription: 'This is a description of your launch'
+      launchDescription: 'This is a description of your launch',
+      launchAttributes: [{ key: 'yourKey', value: 'yourValue' }],
       projectName: 'YOUR_PROJECT',
+      rerun: false,
       debug: false
     }
   ...
 }
 ```
 
 To use this helper you need to provide the following info:
--`token`: which can be found by navigating to the user profile page, clicking the username drop-down in the right header and selecting the "Profile" option > "UUID" – is a unique user identifier. UUID is used in automated test configuration files for a user authentication instead of a password. It will allow you to post data, without logging it in the UI.
--`endpoint`: your reportportal host + `api/v1` for instance: `http://localhost:8080/api/v1`
--`launchName`: the launch name you want, if not provided, the suite title will be used
--`launchDescription`: the description of your launch, if not provided, the description will be empty
--`projectName`: the project that you created in the reportportal UI
--`debug`: to turn on the debug for reportportal
+
+    - `token`: which can be found by navigating to the user profile page, clicking the username drop-down in the right header and selecting the "Profile" / "UUID" – is a unique user identifier. UUID is used in automated test configuration files for a user authentication instead of a password. It will allow you to post data, without logging it in the UI.
+    - `endpoint`: your reportportal host + `api/v1` for instance: `http://localhost:8080/api/v1`
+    - `launchName`: the launch name you want, if not provided, the suite title will be used
+    - `launchDescription`: the description of your launch, if not provided, the description will be empty
+    - `launchAttributes`: the attributes of your launch, if not provided, the attributes will be empty
+    - `projectName`: the project that you created in the reportportal UI
+    - `debug`: to turn on the debug for reportportal
+    - `rerun`: to enable rerun
+    - `rerunOf`: UUID of launch you want to rerun. If not specified, report portal will update the latest launch with the same name
```

Comment:
@AmsterGet have you tried `rerun` without `rerunOf`? I got an error

```sh
(node:64016) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 10)
(node:64016) UnhandledPromiseRejectionWarning: Error: Request failed with status code 404: {"errorCode":4041,"message":"Launch 'This is awesome' not found. Did you use correct Launch ID?"}
    at /Users/thanhnguyen/Desktop/codeceptjs-rest-demo/node_modules/reportportal-client/lib/rest.js:27:23
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:64016) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 11)
```

Reply:
If the `rerunOf` option not specified, items from the launch will be added to the latest launch with the same name (https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md#latest-launch).

Reply:
I see, cause with my previous tests, I removed all launches =))